src/rf/apps/core/management/commands/createawsresources.py

```diff
@@ -5,9 +5,6 @@
 
 from django.core.management.base import BaseCommand
 
-VISIBILITY_TIMEOUT = 120
-MAX_RECEIVE_COUNT = 5
```

Comment:
It's sometimes nice to keep these constants even if they're not used multiple times so that if you want to change tweaky settings, you don't have to dig around through an entire module to find the exact number that is being used. And since code editors handle this well, you can jump to the exact line.